liba/src/strlcat.c

```diff
@@ -0,0 +1,19 @@
+#include <string.h>
+
+size_t strlcat(char * dst, const char * src, size_t dstSize) {
+  const size_t srcLen = strlen(src);
+  size_t dstLen = strlen(dst);
+  if (dstLen > dstSize) {
```

Comment:
Replace these two conditions by
 if (dstLen >= dstSize) {
     return dstSize+srcLen;
 }

Reply:
And this way, dstLen can become const